src/terminal/emoji.rs

```diff
@@ -25,6 +25,6 @@ pub static SLEUTH: Emoji = Emoji("🕵️‍♂️", "");
 pub static SPARKLES: Emoji = Emoji("✨  ", "");
 pub static SWIRL: Emoji = Emoji("🌀 ", "");
 pub static UP: Emoji = Emoji("🆙 ", "");
-pub static WARN: Emoji = Emoji("⛔ ", "");
+pub static WARN: Emoji = Emoji("⚠️", "");
```

Comment:
You need to add a space after the emoji here to match the style of the rest. After that, this looks good to me!